travertino/src/travertino/colors.py

```diff
@@ -345,7 +343,7 @@ def rgba(self):
         elif h < 4.0:
             r, g, b = m, x + m, c + m
         elif h < 5.0:
-            r, g, b = m, x + m, c + m
+            r, g, b = x + m, m, c + m
```

Comment:
I'm not sure why this isn't triggering a test failure. The *change* is correct; but it should be triggering a change in test behavior, and the code *should* have coverage.

Reply:
See #3585 for a full correction here, including extra test cases that reveal the problem. We were testing this - but not on cases that were interesting from the perspective of this bug